### ISSUE TITLE
how i can use it with oh_my_zsh ?

### ISSUE THREAD

**Issue by ysolis** [endaaman/lxd-completion-zsh#2]
well, this is my question, i use oh_my_zsh/zsh in my personal/work laptop and in my servers where i work.

**Comment by endaaman** [comment]
@ysolis 

This completion is not provided as an OMZ(oh-my-zsh) plugin. Originally OMZ is useful but not extensible. So, it is recommended to use OMZ as a one of zsh plugins by introducing plugin-manager such as [Antigen](https://github.com/zsh-users/antigen) or [zplug](https://github.com/zplug/zplug). For example you can write for

```
source /path-to-antigen/antigen.zsh
antigen use oh-my-zsh
antigen bundle endaaman/lxd-completion-zsh
```

or see [my zshrc](https://github.com/endaaman/dotfiles/blob/master/zshrc#L20)


Otherwise, if you do not want to introduce plugin manager, you should follow the `Manually` section in  README.md.

### 1. Clone this repo

```
$ git clone https://github.com/endaaman/lxd-completion-zsh.git /path/to/lxd-completion-zsh
```

Replace `/path/to/lxd-completion-zsh` into your preferred directory e.g. `~/.zsh/completions/lxd`, `~/.zsh/lxd-completion-zsh`, etc.

### 2. Add fpath

To tell to zsh where the completion is, write in `.zshrc` 

```
fpath=(/path/to/lxd-completion-zsh $fpath)
```

This sentence need to be written **before** `source $ZSH/oh-my-zsh.sh` because `fpath` is loaded in the line.

**Comment by ostaszewskik** [comment]
@ysolis, provided that you have default zsh configuration, you can just type (in case of Ubuntu and Debian-based):

`sudo curl -L "https://raw.githubusercontent.com/endaaman/lxd-completion-zsh/master/_lxc" -o /usr/share/zsh/vendor-completions/_lxc`

And open new shell to use autocompletition funcionality.

@endaaman great work! Keep it up!

**Comment by jadjay** [comment]
@mDfRg @endaaman 
Hi, 
I'm really sorry to said that but none of your method works in my case...

**Comment by jadjay** [comment]
@mDfRg @endaaman

It only works when I disabled oh-my-zsh...

**Comment by endaaman** [comment]
@jadjay 

How about executing `rm ~/.zcompdump*` ?

**Comment by ostaszewskik** [comment]
To be honest, it stopped working for a while and @endaaman solution with `rm ~/.zcompdump*` made it rolling again!

**Comment by endaaman** [comment]
Solved #5
